skimage/_shared/_interpolation_test.pyx

```diff
@@ -0,0 +1,31 @@
+from interpolation cimport coord_map as _coord_map
```

Comment:
Would you mind doing the following:

interpolation.pyx -> interpolation_cy.pyx
interpolation.py (wrapper for coord_map)
Make the _interpolation_test.pyx a unit test and move to _shared/tests/test_interpolation.py